#! /bin/sh /usr/share/dpatch/dpatch-run
## 80_security_build.dpatch by Mike Hommey <glandium@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Don't automatically create the .chk files, since we will strip
## DP: the library in debian/rules.
## DP: Also add a dirty hack to load libfreebl from /usr/lib/nss.

@DPATCH@

--- nss/mozilla/security/nss/cmd/shlibsign/Makefile
+++ nss/mozilla/security/nss/cmd/shlibsign/Makefile
@@ -110,5 +110,5 @@
 	$(call core_abspath,$(NSPR_LIB_DIR)) $(call core_abspath,$<)
 endif
 
-libs install :: $(CHECKLOC)
+libs install ::
 
--- nss/mozilla/security/nss/lib/freebl/loader.c
+++ nss/mozilla/security/nss/lib/freebl/loader.c
@@ -45,7 +45,7 @@
 #include "prinit.h"
 
 static const char* default_name =
-    SHLIB_PREFIX"freebl"SHLIB_VERSION"."SHLIB_SUFFIX;
+    "nss/"SHLIB_PREFIX"freebl"SHLIB_VERSION"."SHLIB_SUFFIX;
 
 /* getLibName() returns the name of the library to load. */
 
